template/README.md

```diff
@@ -1,19 +1,29 @@
-# Sanity Demo
+# The Kitchen Sink Studio
 
-This is an example project we use to demonstrate how you can use the Sanity platform.
+A “[kitchen sink](https://stackoverflow.com/questions/33779296/what-is-exact-meaning-of-kitchen-sink-in-programming)” is a showcase of APIs and functionality. This is the example project that we at Sanity HQ for demos. Hence, it will change and evolve over time with new feature releases.
+
+This starter is great for demo purposes, either to evaluate Sanity for your own use, or when you want to offer it as part of a project for your clients.
 
 You can bootstrap a new project from this template by following this link to [sanity.io/create](https://www.sanity.io/create?template=sanity-io%2Fsanity-template-kitchen-sink).
 
 ## What you have
 
-This is a demo example of a customized Sanity Studio. It is indented to be a sort of kitchen sink, a grab bag of many different examples and usages. All of them might not even necessarily make sense to have in a single Sanity Studio, but showing off different concepts and customizations is the goal here. Expect this template to expand and change a lot.
-
-Some concepts covered in this template are
+Concepts covered in this starter are:
 
-- Create landing pages and do page building on them for a web frontend
-- A simple blog with web preview right in the Studio
+- How to use the Studio‘s page builder to create landing pages, and present them in a web frontend
+- Edit blog posts with side-by-side web-previews, can be configured to [Gatsby Preview](https://www.gatsbyjs.com/) for real-time previewing in production and not just local development
+- How to manage SEO metadata like fields for Open Graph meta-tags
+- How to set up navigation for structured content, which allows for proper decoupling and omni-channel
 - Custom embedded objects in rich text for Instagram posts, social media videos, LaTeX math etc.
-- Simple example of Google Optimize A/B test facility
+- [Field validations](https://www.sanity.io/docs/validation)
+- Custom [slugify functions with promises](https://www.sanity.io/docs/slug-type#custom-slugify-function-e1531d9d041b)
+- Examples of custom document and document type lists using [structure builder](https://www.sanity.io/docs/overview-structure-builder)
+- Example of [Google Optimize](https://optimize.google.com/optimize/home/) A/B-test facility
```

Comment:
Google Optimize should have been removed from the README and stay commented out of the schema. I have removed its handling in the gatsby app, and the future plan is to set up and end-to-end working example of a/b testing with Outsmartly. Pending scheduling via Even and Shalom

Reply:
Can‘t we just keep it in and append/replace it when that time comes?

Reply:
Disappointed reactions while demoing at the bare bones level of this way of doing it is the reason I took it out. It was usually a low point of excitement in a demo call.